pkgs/by-name/st/stalwart-mail/package.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, protobuf
+, bzip2
+, openssl
+, sqlite
+, zstd
+, stdenv
+, darwin
+, nix-update-script
+, rocksdb_8_11
+}:
+
+let
+  # FIXME Missing support for rocksdb 9.X
+  # https://github.com/stalwartlabs/mail-server/issues/407
```

Comment:
I suggest adding some links to remember how to find the specific rocksdb version needed by stalwart:
```suggestion
# Stalwart depends on rocksdb crate:
# https://github.com/stalwartlabs/mail-server/blob/v0.8.0/crates/store/Cargo.toml#L10
# which expects a specific rocksdb versions:
# https://github.com/rust-rocksdb/rust-rocksdb/blob/v0.22.0/librocksdb-sys/Cargo.toml#L3 
```